.github/workflows/npm_publish.yml

```diff
@@ -23,7 +23,16 @@ jobs:
       - name: Build, Test, and Package
         run: yarn preversion
 
-      - name: Publish npm package
-        run: yarn publish --tag beta
+      - name: Publish npm package to both places
+        run: |
+          yarn publish --access public
+          sed -i -e 's#"@stellar/stellar-sdk"#"stellar-sdk"#' package.json
```

Comment:
wow, this works, nice!

Reply:
yep heh, tried it out in https://github.com/stellar/js-xdr/pull/111 and it works well :+1: